pkg/ipmi/structures.go

```diff
@@ -0,0 +1,128 @@
+// Copyright 2022 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package ipmi implements functions to communicate with the OpenIPMI driver
```

Comment:
The package comment shall only be in one file per package

Reply:
Whoopsi.